pkg/csi/csi_ops.go

```diff
@@ -187,14 +199,92 @@ func (c *applicationCreate) CreatePod(ctx context.Context, args *types.CreatePod
 	return podRes, nil
 }
 
+func (c *applicationCreate) WaitForPVCReady(ctx context.Context, namespace, name string) error {
+	if c.kubeCli == nil {
+		return fmt.Errorf("kubeCli not initialized")
+	}
+
+	err := c.waitForPVCReady(ctx, namespace, name)
+	if err != nil && (errors.Is(err, context.DeadlineExceeded) || strings.Contains(err.Error(), "would exceed context deadline")) {
```

Comment:
Can we just do `if err != nil` here? I don't see why we need to single out `context.DeadlineExceeded`, nor the string comparison.